docs/src/developer/standards.rst

```diff
@@ -1,47 +1,86 @@
-.. contents:: Developer's Guide 103: Contribution Standards
+***********
+Conventions
+***********
 
-**********
-Contribute
-**********
+In this chapter, we present the different standards and guidelines we use throughout the project.
+All the conventions are enforced automatically during the PR process.
 
-Coding and Repository Standards
-===============================
-
-We are using flake8_ (along with some of its plugins) and pylint_.
-Their styles are provided in ``/tox.ini`` and ``/.pylintrc`` respectively.
+You can verify if your code will pass the checks locally beforehand using:
 
 .. code-block:: sh
 
-   tox -e flake8
-   tox -e pylint
+   tox -e lint
```

Comment:
Maybe add something like _which is equivalent to `tox -e flake8,pylint,doc8,packaging`. Otherwise it may be confusing to contributors since `lint` is not used in the CI.